package.json

```diff
@@ -33,6 +33,7 @@
     "watch:lessToSass": "grunt watch:lessToSass"
   },
   "dependencies": {
+    "@splidejs/splide": "^4.1.4",
```

Comment:
Based on conversation elsewhere with @EreMaijala, I think this should get moved down to the devDependencies section, since it is required for building VuFind but not for running VuFind.

Reply:
We're having this conversation in multiple PRs. Let's try to remember to talk this out on the next Community Call so we can put it to rest one way or another. :-)